src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -22,9 +19,8 @@ private static BoundStatement AddSequencePoint(BoundStatement node)
 
         internal static BoundStatement AddSequencePoint(VariableDeclaratorSyntax declaratorSyntax, BoundStatement rewrittenStatement)
         {
-            SyntaxNode node;
             TextSpan? part;
-            GetBreakpointSpan(declaratorSyntax, out node, out part);
+            GetBreakpointSpan(declaratorSyntax, out _, out part);
```

Comment:
can't we use `out var` here?

Reply:
Constructors must have a body in syntactically correct tree.